---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---


{% if author.googlescholar %}
  You can also find my articles on <u><a href="{{author.googlescholar}}">my Google Scholar profile</a>.</u>
{% endif %}

*Sorted by domains and years.*

### Methods in Statistical Genetics and Genomics
* **Sun, J.**, Dong Q., Wei, J., Gao, Y., Yu, Z., Hu, X., Zhang, Y. ti-scMR: Trajectory-inference-based dynamic single-cell Mendelian randomization identifies causal genes underlying phenotypic differences. *Under Review*. <br>
[**Codes**](https://github.com/sjl-sjtu/ti-scMR)

*	**Sun, J.**, Zhou, J., Gong, Y., Pang, C., Ma, Y., Zhao, J., Yu, Z., & Zhang, Y. (2024). Bayesian network-based Mendelian randomization for variant prioritization and phenotypic causal inference. *Human Genetics*, 143(9-10), 1081â€“1094. <br>
[**Paper**](https://doi.org/10.1007/s00439-024-02640-x) | [**PDF**](../files/BNMR.pdf) | [**Supplementary materials**](https://static-content.springer.com/esm/art%3A10.1007%2Fs00439-024-02640-x/MediaObjects/439_2024_2640_MOESM1_ESM.pdf) | [**R package**](https://github.com/sjl-sjtu/bnmr2)

*	Gong, Y., Xu, J., Wu, M., Gao, R., **Sun, J.**, Yu, Z., & Zhang, Y. (2024). Single-Cell Biclustering for Cell-Specific Transcriptomic Perturbation Detection in AD Progression. *Cell Reports Methods*, 4(4), 100742. <br>
[**Paper**](https://doi.org/10.1016/j.crmeth.2024.100742) | [**PDF**](../files/scBC.pdf) | [**Python package**](https://github.com/sjl-sjtu/scBC)

*	**Sun, J.**, Lyu, R., Deng, L., Li, Q., Zhao, Y., & Zhang, Y. (2022). SMetABF: A rapid algorithm for Bayesian GWAS meta-analysis with a large number of studies included. *PLOS Computational Biology*, 18(3), e1009948. <br>
[**Paper**](https://doi.org/10.1371/journal.pcbi.1009948) | [**PDF**](../files/SMetABF.pdf) | [**R package**](https://github.com/sjl-sjtu/GWAS_meta) | [**Online tool**](https://sunjianle-sjtu.shinyapps.io/analycode/)

*	Lyu, R., **Sun, J.**, Xu, D., Jiang, Q., Wei, C., & Zhang, Y. (2021). GESLM algorithm for detecting causal SNPs in GWAS with multiple phenotypes. *Briefings in Bioinformatics*, 22(6), bbab276. <br>
[**Paper**](https://doi.org/10.1093/bib/bbab276) | [**PDF**](../files/GESLM.pdf) | [**R package**](https://github.com/sjl-sjtu/GESLM)

*	Zhou, Y., Fa, B., Wei, T., **Sun, J.**, Yu, Z., & Zhang, Y. (2021). Elastic Correlation Adjusted Regression (ECAR) scores for high dimensional variable importance measuring. *Scientific Reports*, 11(1), 23354. <br>
[**Paper**](https://doi.org/10.1038/s41598-021-02706-0) | [**PDF**](../files/ECAR.pdf)

### Causal Inference and Machine Learning
*	Dai, H., Ng, I., **Sun, J.**, Tang, Z, Luo, G., Dong, X., Spirtes, P., Zhang, K. (2025) When Selection meets
Intervention: Additional Complexities in Causal Discovery. *The Thirteenth International Conference on Learning Representations (ICLR)*, (Oral).

### Epidemiology
*	**Sun, J.**, Deng, L., Li, Q., Zhou, J., & Zhang, Y. (2024). Dynamic relations between longitudinal morphological, behavioral, and emotional indicators and cognitive impairment: evidence from the Chinese Longitudinal Healthy Longevity Survey. *BMC Public Health*, 24(1), 3516. <br>
[**Paper**](https://doi.org/10.1186/s12889-024-21072-w) | [**PDF**](../files/CLHLS.pdf) | [**Supplementary materials**](https://static-content.springer.com/esm/art%3A10.1186%2Fs12889-024-21072-w/MediaObjects/12889_2024_21072_MOESM1_ESM.pdf) | [**Codes**](https://github.com/sjl-sjtu/CLHLS)

* **Sun, J.**, Deng, L., Zhu, H., Liu, M., Lyu, R., Lai, Q., & Zhang, Y. (2021). Meta-analysis on the association between rs11868035, rs823144, rs3851179 and Parkinson's disease. *Meta Gene*, 30, 100949.

### Foundation Models in Biology
* Liang, C. $\#$, Ye, P.$\#$, Yan, H.$\dag$, Zheng, P$\dag$, **Sun, J.**, Wang, Y., Li, Y., Ren, Y., Jiang, Y., Xiang, J., Zhang, S., Jiang, L., Bai, W., Ma, X., Chen, T., Zuo, W., Bai, L., Ouyang, W., Li, J. (2025). scWGBS-GPT: A Foundation Model for Capturing Long-Range CpG Dependencies in Single-Cell Whole-Genome Bisulfite Sequencing to Enhance Epigenetic Analysis. *bioRxiv*. <br>
[**bioRxiv**](https://www.biorxiv.org/content/10.1101/2025.02.19.638959v1)

* Ye, P.$\dag$, Bai, W.$\dag$, Ren, Y.$\dag$, Li, W.$\dag$, Qiao, L., Liang, C., Wang, L., Cai, Y., **Sun, J.**, Yang, Z., Zheng, P., Dong, N., Chen, T., Wang, Z., Liu, X., Ma, X., Yan, H., Wang, Z., Wang, S. & Ouyang, W. (2024). Genomics-FM: Universal Foundation Model for Versatile and Data-Efficient Functional Genomic Analysis. *bioRxiv*. <br>
[**bioRxiv**](https://www.biorxiv.org/content/10.1101/2024.07.16.603653v1) | [**Review**](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=5030819)

*	Liang, C., Qiao, L., Ye, P., Dong, N., **Sun, J.**, Bai, W., Ren, Y., Ma, X., Yan, H., Song, C., Ouyang, W., & Zuo, W. (2023). Toward Understanding BERT-Like Pre-Training for DNA Foundation Models. *arXiv*. <br>
[**arXiv**](https://arxiv.org/abs/2310.07644)

You can also find my articles on [Google Scholars](https://scholar.google.com/citations?user=sRFyIxAAAAAJ&hl=en&authuser=1), [ResearchGate](https://www.researchgate.net/profile/Jianle-Sun) or [ORCID](https://orcid.org/0000-0002-0001-0992).
